temporian/implementation/pandas/operators/sum.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2021 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Implementation for the sum operator."""
+
+from temporian.implementation.pandas.data.event import PandasEvent
+from temporian.implementation.pandas.operators.base import PandasOperator
+
+RESOLUTIONS = ["PER_FEATURE_IDX", "PER_FEATURE_NAME"]
```

Comment:
Making this an enum could make it easier for documentation purposes and for the user to see their available choices when calling the operator. Defining it in core would be best to also use it there.

Reply:
Also, I think making the default `PER_FEATURE_IDX` instead of `None` more accurately represents the operator's behavior

Reply:
Yeah, I thought about making an enum but didn't know well where to place it. I will try to create one in core/sum